sljit_src/sljitExecAllocator.c

```diff
@@ -168,6 +168,28 @@ static SLJIT_INLINE void apple_update_wx_flags(sljit_s32 enable_exec)
 #define SLJIT_MAP_JIT	(0)
 #endif /* !SLJIT_MAP_JIT */
 
+#ifdef __FreeBSD__
+#include <sys/param.h>
+#if __FreeBSD_version >= 1301000
```

Comment:
using PROC_WXMAP_CTL directly would be preferably and avoid hardcoding a specific OS version